notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb

```diff
@@ -583,32 +511,21 @@
       "source": [
         "#### Set machine type\n",
         "\n",
-        "Next, set the machine type to use for prediction.\n",
+        "Next, set the machine type to use for deployment.\n",
         "\n",
-        "- Set the variable `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you will use for for training.\n",
-        " - `machine type`\n",
-        "     - `n1-standard`: 3.75GB of memory per vCPU.\n",
+        "- Set the variable `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you use for deployment.\n",
```

Comment:
Remove extra space between "configure" and "the":

Set the variable `DEPLOY_COMPUTE` to configure the compute resources for the VMs you use for deployment.